management/admin/classes/common/DBService.php

```diff
@@ -45,47 +45,53 @@ protected function connect() {
 		$host = $this->config->database->host;
 		$username = $this->config->database->username;
 		$password = $this->config->database->password;
-		$this->db = mysql_connect($host, $username, $password);
+		$this->db = new mysqli($host, $username, $password);
 		$this->checkForError();
 
 		$databaseName = $this->config->database->name;
-		mysql_select_db($databaseName, $this->db);
-		mysql_set_charset('utf8', $this->db);
+		$this->db->select_db($databaseName);
+		$this->db->set_charset('utf8');
 		$this->checkForError();
 	}
 
 	protected function disconnect() {
-		//mysql_close($this->db);
+		//mysqli_close($this->db);
 	}
 
 	public function escapeString($value) {
-		return mysql_real_escape_string($value);
+		return $this->db->escapeString($value);
```

Comment:
^^^^^^^^^^
That was fixed in a latter commit.
https://github.com/Coral-erm/Coral/commit/893abdb5d79e14d82283084ff20d436203c2f9ff#diff-5cf75f5d32772b1eb7e42cb21acede85L61